core/backend/src/CloudSqlite.ts

```diff
@@ -48,6 +48,10 @@ export namespace CloudSqlite {
     writeable?: boolean;
     /** if true, container is attached in "secure" mode (blocks are encrypted). Only supported in daemon mode. */
     secure?: boolean;
+    /** An id which enhances logging provided by CloudSQLite.
+     * This ID will be used to identify, in log messages, all CloudSQLite client connections (also known as database connections) opened using this CloudContainer.
```

Comment:
I assume this only applies in daemon mode? BTW, we always use "Id" rather than "ID".

Reply:
Changed it to Id.

A clientIdentifier could be provided in the daemonless case, it probably just wouldn't be as useful. BCV_HTTP_LOG table would still exist in daemonless mode (and uses clientIdentifier), but I believe we only really care about it in the daemon case.

Reply:
i think you should document that as only being relevant for daemon mode. It's confusing to explain why you need to identify yourself to yourself without a daemon.

Reply:
"This Id is mostly only relevant to give more clarity to logs produced running in daemon mode, where there are usually many active CloudContainers and by extension, many ongoing HTTP requests."

What do you think of the above?